docker/release/base/Dockerfile.in

```diff
@@ -2,9 +2,10 @@ FROM fedora:37 AS keylime_base
 LABEL version="_version_" description="Keylime Base - Only used as an base image for derived packages"
 MAINTAINER Keylime Team <main@keylime.groups.io>
 
-RUN dnf -y install dnf-plugins-core git && dnf -y builddep tpm2-tools
-RUN git clone -b 5.4 https://github.com/tpm2-software/tpm2-tools.git && \
+RUN dnf -y install dnf-plugins-core git efivar-libs efivar-devel && dnf -y builddep tpm2-tools 
+RUN git clone -b master https://github.com/tpm2-software/tpm2-tools.git && \
     cd tpm2-tools && \
+    git checkout 1529b0bf99f4b41608770b011effa5bda35a3976 && \
```

Comment:
... use cherry-pick of commits 576a31bcc910da517067b29667f45fbe78e812e0, b2b8b1aa81ef298d33126ac74dcd7590b6e36515 and 1529b0bf99f4b41608770b011effa5bda35a3976, that I guess are the one you are interested it.

The reason is that master can be fairly broken some days, making it to debug a issue. It is better to control what is inside the container.